plugins/discourse-assign/plugin.rb

```diff
@@ -899,11 +899,11 @@ module ::DiscourseAssign
     next if names.blank?
 
     if names.include?("nobody")
-      next scope.where("topics.id NOT IN (SELECT a.topic_id FROM assignments a WHERE a.active)")
+      next scope.where("topics.id NOT IN (#{Assignment.listable.select(:topic_id).to_sql})")
```

Comment:
I believe if we use string interpolation ActiveRecord won't use a prepared statement, and there's some tiny risk of SQL injection in the future.

```suggestion
      next scope.where("topics.id NOT IN (?)", Assignment.listable.select(:topic_id))
```